core/src/internal/image_utils.cpp

```
@@ -117,6 +117,9 @@ void invert_image(uint8_t* ptr, int width, int height) {
 }
 
 void threshold_image(uint8_t* ptr, const int width, const int height, const int num_thresholds) {
```

Cppcheck (2.21.0)

[style] The function 'threshold_image' is never used.

(unusedFunction)